my_project/tab_psy_chart/app_psy_chart.py

```diff
@@ -215,6 +216,7 @@ def layout_psy_chart():
 @app.callback(
     Output("psych-chart", "children"),
     [
+        Input("df-store", "modified_timestamp"),
```

Comment:
Are you using this ts information?

Reply:
This is needed. Because when the user changes the si-ip state on this page, all the data and graphs need to change, so ts information is used to listen for the changes.